tests/test_annotation.py

```diff
@@ -302,3 +302,19 @@ def test_polygon_gt_upload_ignore(dataset):
     assert_polygon_annotation_matches_dict(
         response_annotation, TEST_POLYGON_ANNOTATIONS[0]
     )
+
+    @pytest.mark.integration
```

Comment:
way to catch this detail. helps a lot for local development to be able to skip these tests that spin up step functions